packages/cadl-rlc-test/spec/main.cadl

```diff
@@ -1,208 +1,339 @@
 import "@cadl-lang/rest";
 import "@cadl-lang/versioning";
 import "@azure-tools/cadl-azure-core";
-import "@azure-tools/cadl-autorest";
 
 using Cadl.Http;
 using Cadl.Rest;
 using Cadl.Versioning;
 using Azure.Core;
-using OpenAPI;
 
-//
-// ** Service description **
-//
+// NOTE: These features are missing:
+// - Security definition for apiKey
 
-// TODO: Use `Uri` type when available: https://github.com/microsoft/cadl/issues/654
-@serviceTitle("Confidential Ledger Service")
-@serviceVersion("2022-05-13")
-@useAuth(OAuth2Auth<[MyFlow]>)
-@versionedDependency(Azure.Core.Versions.v1_0_Preview_1)
 @server(
-  "{ledgerUri}",
-  "Confidential Ledger Service",
```

Comment:
I think we could keep the `Confidential Ledger Service` as our smoke test. So we could guarantee the changes will not break that RP.

I could help add into smoke test if necessary.

Reply:
Maybe we should move them both to the smoke-test and remove the cadl-rlc-test folder @joheredi how do you think of it ?